game-core/src/main/java/games/strategy/engine/data/RouteScripted.java

```diff
@@ -14,8 +14,6 @@
 public class RouteScripted extends Route {
   private static final long serialVersionUID = 604474811874966546L;
 
-  public RouteScripted() {}
```

Comment:
I mixed scopes here a bit, the two constructors removed are unused.

Side note: would be great to see this class combined with `Route.java`